test/claim-service.test.js

```diff
@@ -0,0 +1,73 @@
+
+describe('Test claim service', () => {
+  let claimService
+  let claimRepositoryMock
+  let minetypeRepositoryMock
+  let messageServiceMock
+
+  beforeEach(async () => {
+    claimRepositoryMock = require('../server/repository/claim-repository')
+    minetypeRepositoryMock = require('../server/repository/minetype-repository')
+    messageServiceMock = require('../server/services/message-service')

```

Comment:
Do these `require` statements achieve anything, given that we mock each of the modules immediately below?

If we do need to require modules before mocking them, the `require` statements should be at the top of the file so they're easy to see and only called once.